shared/src/main/java/com/couchbase/lite/Blob.java

```diff
@@ -535,7 +535,7 @@ public int read(byte[] b) throws IOException {
             try {
                 byte[] bytes = readStream.read(b.length);
```

Comment:
Would in be  better to `if (bytes.length <= 0) { return -1; }`, right here, before the copy?